invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js

```diff
@@ -15,39 +11,49 @@ import { default as config } from './config';
 import { goTo } from '../../../history';
 
 export default class Home extends Component {
-  _renderSearchBar = (_, queryString, onInputChange, executeSearch) => {
-    const onBtnSearchClick = (event, input) => {
-      executeSearch();
-      goTo(FrontSiteRoutes.documentsListWithQuery(queryString));
-    };
+  constructor(props) {
+    super(props);
+    this.state = { query: '' };
+    this.updateQuery = this.updateQuery.bind(this);
+  }
+
+  updateQuery(event) {
+    this.setState({ query: event.target.value });
+  }
 
-    return (
-      <DocumentsSearchBar
-        currentQueryString={queryString}
-        onInputChange={onInputChange}
-        executeSearch={onBtnSearchClick}
-        placeholder={'Search for books, articles, proceedings...'}
-        queryHelperFields={config.HELPER_FIELDS}
-      />
-    );
-  };
   render() {
     return (
       <div className="home-container">
         <Grid centered columns={2}>
           <Header size="huge">What would you like to find?</Header>
 
           <Grid.Row centered columns={2}>
-            <ReactSearchKit
-              searchConfig={{
-                ...apiConfig,
-                url: documentApi.url,
-              }}
-            >
-              <Container className="books-search-searchbar">
-                <SearchBar renderElement={this._renderSearchBar} />
-              </Container>
-            </ReactSearchKit>
+            <Container className="books-search-searchbar">
+              <Form
+                onSubmit={goTo(
```

Comment:
Is `onSubmit` needed here when there is an `onClick` in the icon link?

Reply:
I think is for the enter event but still there is an event for that I think...

Reply:
Indeed, it is

Reply:
Yes, you're right, I forgot about that!